webextensions/api/webRequest.json

```diff
@@ -117,18 +117,29 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/RequestFilter",
             "support": {
               "chrome": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
                 "version_added": true
               },
               "edge": {
+                "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
                 "version_added": "14"
               },
-              "firefox": {
-                "version_added": "45"
-              },
+              "firefox": [
+                {
+                  "version_added": "45"
+                },
+                {
+                  "notes": "If a filter contains unrecognized values in its <code>types</code> property, this is treated as an error.",
+                  "version_added": "45",
+                  "version_removed": "78"
```

Comment:
@wbamberg`version_removed` makes me think that ResourceType isn't supported anymore in 78, like if we removed it completely, but it is still supported in "Firefox > 78" and it just changed its behavior in case of unsupported values.

is there a way to express this without making it looks like we removed it in that version?

I see that in the userScripts.json to express that the API was originally locked behind a pref we do use to version_added entries one with a flags property and one without it: 
https://github.com/mdn/browser-compat-data/blob/790eeb364cf5f19e589830b5242547890fbf7039/webextensions/api/userScripts.json#L16-L28

and so I'm wondering if it should be ok to use to version_added entries, and one with version_added and the note?
To be fair this change sounds more like a "version_changed" entry, but I think that a "version_changed" property is not in the schema we use for this metadata, at least not at the moment.